Formula/fypp.rb

```diff
@@ -0,0 +1,32 @@
+class Fypp < Formula
+  include Language::Python::Virtualenv
+
+  desc "Python powered Fortran preprocessor"
+  homepage "https://fypp.readthedocs.io/en/stable/"
+  url "https://github.com/aradi/fypp/archive/refs/tags/3.1.tar.gz"
+  sha256 "0f66e849869632978a8a0623ee510bb860a74004fdabfbfb542656c6c1a7eb5a"
+  license "BSD-2-Clause"
+  head "https://github.com/aradi/fypp.git", branch: "master"
+
+  depends_on "gcc" => :test
+  depends_on "python@3.9"
```

Comment:
Does it work with `3.10`?

Reply:
Sure, should this be bumped to 3.10 or is there something like a version range available?

Reply:
This should be bumped to 3.10, all formulae should use the latest dependencies that work.